app/sidekiq/lighthouse/pension_benefit_intake_job.rb

```diff
@@ -113,11 +113,26 @@ def check_success(response)
       if response.success?
         Rails.logger.info('Lighthouse::PensionBenefitIntakeJob Succeeded!', { saved_claim_id: @claim.id })
         @claim.send_confirmation_email if @claim.respond_to?(:send_confirmation_email)
+        form_submission_polling
       else
         raise PensionBenefitIntakeError, response.to_s
       end
     end
 
+    # Insert submission polling entries
+    def form_submission_polling
+      form_submission = FormSubmission.create(
+        form_type: @claim.form_id,
+        form_data: @claim.to_json,
```

Comment:
Could this potentially expose veteran's PII?

Reply:
Not with the changes here.  The submission or attempt should not be logged (and are not at the moment).

Reply:
ok, sounds good!